src/main/java/webserver/request/HTTP_HEADERS.java

```diff
@@ -1,6 +1,6 @@
 package webserver.request;
 
-public class HTTP {
+public class HTTP_HEADERS {
```

Comment:
Java folks really want you to call this `HttpHeaders`, even if all it contains is constants.  You could make an `enum` if you like, but I wouldn't hold up this PR over it.

An enum would look something like this

```java
public class HttpHeader {
  OK(200, "OK"),
  CREATED(201, "Created"),
  ...

  private HttpHeader(int statusCode, String statusText) {
    this.statusCode = ...
    this.statusText = ...
  }
  
  //Note the object-oriented design here.  This method *does a thing* instead of simply exposing its data.
  public void writeStatusTo(PrintWriter writer) P{
    writer.println(String.format("HTTP/1.1 %s %s", this.statusCode, this.statusText);
  }
}

//Using the enum
HttpHeader.OK.writeTo(new PrintWriter(socket));
```

Reply:
Is it ok to just send the initial line? Not all together with the body?

Reply:
You could use one class or function to send the first line and a different class or function to send the rest of the content if you like.